db/schema.rb

```diff
@@ -126,7 +126,6 @@
   end
 
   add_index "notification_acknowledgements", ["notification_id"], name: "index_notification_acknowledgements_on_notification_id", using: :btree
-  add_index "notification_acknowledgements", ["user_id"], name: "index_notification_acknowledgements_on_user_id", using: :btree
```

Comment:
We shouldn't remove this index (not sure why the change in this commit would do that though)